src/Mutator/Operator/Continue_.php

```diff
@@ -77,12 +77,8 @@ public function canMutate(Node $node): bool
             return false;
         }
 
-        $parentNode = $node->getAttribute(ParentConnectorVisitor::PARENT_KEY);
+        $parentNode = ParentConnector::findParent($node);
 
-        if ($parentNode instanceof Node\Stmt\Case_) {
-            return false;
-        }
-
-        return true;
+        return !($parentNode instanceof Node\Stmt\Case_);
```

Comment:
Are parentheses required here? It was a thing in older versions of PHP IIRC.

Reply:
they are not, it's just clearer IMO. But might just be me

Reply:
As of current master, we have about 37 cases where parentheses are not used in a similar situation, and only 6 where are.

Reply:
I'll do it in another PR